core/domain/feedback_services.py

```diff
@@ -1067,9 +1069,8 @@ def _send_instant_emails(
     for recipient_id, can_receive_email in python_utils.ZIP(
             recipient_list, can_recipients_receive_email):
         if can_receive_email:
-            transaction_services.run_in_transaction(
-                enqueue_feedback_message_instant_email_task, recipient_id,
-                feedback_message_reference)
+            enqueue_feedback_message_instant_email_task(
```

Comment:
Per consistency with other methods--should this be named ``enqueue_feedback_message_instant_email_task_transactional``?

Reply:
Yes @BenHenning , but pylint tests are failing in `feedback_services_test.py` as it exceeds 80 characters, So I tried to put it in second-line but it was not working, `feedback_services.enqueue_feedback_message_instant_email_task_transactional`

Reply:
You can call it like this in the tests
```
(
    feedback_services
    .enqueue_feedback_message_instant_email_task_transactional(
        enqueue_feedback_message_instant_email_task, 
        recipient_id,
        feedback_message_reference
    )
)
```

Reply:
@vojtechjelinek Yes, but it's giving syntax error here `.enqueue_feedback_message_instant_email_task_transactional`. I tried to do that before.
Thanks

Reply:
Could you please send which exact arror is this throwing?

Reply:
@vojtechjelinek Here  `E:784, 0: invalid syntax (<string>, line 784) `
![Screenshot from 2021-02-06 21-00-41](https://user-images.githubusercontent.com/44300735/107122722-6f488d00-68bf-11eb-9ef0-8f75c0fc1db5.png)

This is `feedback_services_test.py` as I changed the function name in `feedback_services.py`
Thanks

Reply:
You need to have around brackets around the code (see the example that I sent).

Reply:
Forgot that, Thanks.